lms/envs/aws.py

```diff
@@ -274,6 +274,16 @@
     COMPREHENSIVE_THEME_DIR = ENV_TOKENS.get('COMPREHENSIVE_THEME_DIR')
 
 COMPREHENSIVE_THEME_DIRS = ENV_TOKENS.get('COMPREHENSIVE_THEME_DIRS', COMPREHENSIVE_THEME_DIRS) or []
+
+# THEME_LOCALE_PATHS contain the paths to themes locale directories e.g.
+# "THEME_LOCALE_PATHS" : [
+#        "/edx/src/edx-themes/conf/locale"
+#    ],
+THEME_LOCALE_PATHS = ENV_TOKENS.get('THEME_LOCALE_PATHS', [])
```

Comment:
I know the name is long, but we should probably keep to the "comprehensive" convention and go with `COMPREHENSIVE_THEME_LOCALE_PATHS` here